### PR TITLE
deleteAccount() and bugfixes

### PR DESCRIPTION
**Pull request by DenisIvanov26**

### CODE REVIEW

**Review by skunal8197 — APPROVED** [review]
Changes for deleteAccount look good. The rest of the changes are picked up by rebasing from main

**Review by alanorwick — CHANGES_REQUESTED** [review]
Need to fix linter.

**Review by alanorwick — CHANGES_REQUESTED** [review]

**Review by skunal8197 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -135,6 +138,23 @@ <h1>Tests</h1>
...
+      const response = await ethereum.request({
+        method: 'wallet_invokeSnap',
+        params: [snapId, {
+          method: 'deleteAccount',
+          params: {
+            address: document.getElementById('deleteAccount').value
+          }
+        }]
+      })
+      console.log(response)
```
Comment (alanorwick) [inline index.html:152]: still logging response

**Review by alanorwick — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (DenisIvanov26) [reply index.html]: Remopved all logging statements

**Review by alanorwick — APPROVED** [review]

**Pull request merged**